### ISSUE TITLE
rules_closure problem: function change

### ISSUE THREAD

**Issue by kashiwachen** [jetsonhacks/installTensorFlowTX2#23]
nvidia@tegra-ubuntu:/media/nvidia/YYFSD/DL/installTensorFlowTX2$ ./buildTensorFlow.sh 
ERROR: /home/nvidia/.cache/bazel/_bazel_nvidia/d2751a49dacf4cb14a513ec663770624/external/io_bazel_rules_closure/closure/stylesheets/closure_css_binary.bzl:65:13: The function 'set' has been removed in favor of 'depset', please use the latter. You can temporarily refer to the old 'set' constructor from unexecuted code by using --incompatible_disallow_uncalled_set_constructor=false
ERROR: error loading package '': Extension 'closure/stylesheets/closure_css_binary.bzl' has errors
ERROR: error loading package '': Extension 'closure/stylesheets/closure_css_binary.bzl' has errors
INFO: Elapsed time: 0.284s
FAILED: Build did NOT complete successfully (0 packages loaded)

**Comment by toreal** [comment]
I have the same problem. It looks like this situation happened was because of  the Bazel version be updated to 0.10.0.
[Bazel version](https://github.com/tensorflow/tensorflow/issues/13377)

If I downgrade Bazel, do I will face the [issue 19 ](https://github.com/jetsonhacks/installTensorFlowTX2/issues/19) .

**Comment by jasonjiang21** [comment]
@toreal @PochiaOscar have you guys solved this issue?

**Comment by toreal** [comment]
@jiangyonghui  Yes. I solved it. I try to get the newest version for all packages.
